src/textarea/index.tsx

```diff
@@ -47,7 +43,7 @@ const StyledTextarea = styled(Textarea)`
     line-height: ${font.base.lineHeight};
     width: 100%;
     caret-color: ${color.inputCaret};
-    padding: ${space.l};
+    padding: calc(${space.l} + 1px - ${inputBorderSize.focus});
```

Comment:
I imagine that `1px` = `${inputBorderSize.default}`

Reply:
yes, but it's not really related to inputBorderSize.default anymore; Only a magic offset to pixel-perfect vertical align.

Reply:
Maybe add a new variable named pixelPerfectMagic then :-)
Nah that's ok